src/diffusers/pipelines/ltx/pipeline_ltx_video2video.py

```diff
@@ -0,0 +1,860 @@
+        )
+        shape = (batch_size, num_channels_latents, num_frames, height, width)
+        mask_shape = (batch_size, 1, num_frames, height, width)
+
+        if latents is not None:
+            conditioning_mask = latents.new_ones(shape)
+            conditioning_mask = self._pack_latents(
+                conditioning_mask, self.transformer_spatial_patch_size, self.transformer_temporal_patch_size
+            )
+            return latents.to(device=device, dtype=dtype), conditioning_mask
+
+        if isinstance(generator, list):
+            if len(generator) != batch_size:
+                raise ValueError(
+                    f"You have passed a list of generators of length {len(generator)}, but requested an effective batch"
+                    f" size of {batch_size}. Make sure the batch size matches the length of the generators."
+                )
+
+            init_latents = [
+                retrieve_latents(self.vae.encode(video[i].unsqueeze(0).permute(0, 2, 1, 3, 4)), generator[i])
+                for i in range(batch_size)
+            ]
+        else: # `premute()` because we want `batch_size, num_channels, num_frames, height, width`
+            init_latents = [
+                retrieve_latents(self.vae.encode(vid.unsqueeze(0).permute(0, 2, 1, 3, 4)), generator) for vid in video
+            ]
+
+        init_latents = torch.cat(init_latents, dim=0).to(dtype)
+        init_latents = self._normalize_latents(init_latents, self.vae.latents_mean, self.vae.latents_std)
+        # `ones()` as we want to condition on all?
+        conditioning_mask = torch.ones(mask_shape, device=device, dtype=dtype)
+
+        noise = randn_tensor(shape, generator=generator, device=device, dtype=dtype)
+        # TODO: consider adding the noise w.r.t the flow equation? CogVideoX vid2vid
+        # adds this noise with `add_noise()`.
+        latents = (1 - sigma) * init_latents + sigma * noise
+        # latents = init_latents * conditioning_mask + noise * (1 - conditioning_mask)
+
+        conditioning_mask = self._pack_latents(
+            conditioning_mask, self.transformer_spatial_patch_size, self.transformer_temporal_patch_size
+        ).squeeze(-1)
+        latents = self._pack_latents(
+            latents, self.transformer_spatial_patch_size, self.transformer_temporal_patch_size
+        )
+
+        return latents, conditioning_mask
+
+    @property
+    def guidance_scale(self):
+        return self._guidance_scale
+
+    @property
+    def do_classifier_free_guidance(self):
+        return self._guidance_scale > 1.0
+
+    @property
+    def num_timesteps(self):
+        return self._num_timesteps
+
+    @property
+    def attention_kwargs(self):
+        return self._attention_kwargs
+
+    @property
+    def interrupt(self):
+        return self._interrupt
+
+    @torch.no_grad()
+    @replace_example_docstring(EXAMPLE_DOC_STRING)
+    def __call__(
```

Comment:
I don't see a `strength` parameter here. Without strength, it is not possible to control how much effect of the original video you would like to have in the output video. Low strength -> low denoising steps -> more similar video. And high strength -> more denoising steps -> less similar video. This is more of a naive approach that we use for vid-to-vid, but with techniques like RF-inversion and FlowEdit (which we'll directly add in modular diffusers instead of pipelines), the quality and possibilities are endless!